### DISCUSSION TITLE
Libre 2+

### DISCUSSION THREAD

**Discussion by Diabeticliberty** [NightscoutFoundation/xDrip#3773] [Q&A] [answered]
Well folks I just got wind that Abbott are phasing out Libre 2 and replacing it with Libre 2+. Their Website is extremely vague about what the new sensor is intended to do or why they are phasing the new one in. I must add though that it doesn't take a genius to work it out. Does anybody know what it will involve? More importantly will Xdrip+ continue to work with it and will the original version of OOP2 continue to translate? Over to all of you from all of me.

**Comment by parapenT1sta** [comment]
Libre 2+ = 15 Days
Libre 2 = 14 Days

**Reply by Diabeticliberty** [reply]
Thank you for your reply. I was actually aware of the extra day of lifespan that the new sensor brings. I am more concerned about whether the new units will work with Xdrip+ and the OOP2 translator. For my own purposes although by no means perfect Xdrip+ has been an absolute game changer in diabetes management. This also applies to a lot of other type 1's who I have recommended Xdrip+ to. Some love it and some just like it. One thing on which we all seem to agree is that Librelink is absolutely dreadful.

**Reply by parapenT1sta** [reply]
Maybe @tzachi-dar can answer regarding OOP2.

**Reply by PhilTU** [reply, marked as answer]
> new units will work with Xdrip+ and the OOP2 translator

since 31/5/24 the OOP2 and xDrip+ versions can handle the UK / EU Libre 2+

The Latin American, Russian US etc L2+ are not supported.

**Reply by Diabeticliberty** [reply]
Many thanks for your reply. I have yet to speak to anyone who used Libre 2+ but am cautious of the corporate silliness of Abbott

**Reply by PhilTU** [reply]
> I have yet to speak to anyone who used Libre 2+

I used one, somewhere on here is the discussion at the time (20/5-31/5) where tzachi-dar made changes to accommodate the new sensor code of the 2+ and i tested it.

**Reply by Diabeticliberty** [reply]
Could I ask what you made of it please. Good, bad or indifferent?

**Reply by PhilTU** [reply]
Same, with an extra day.

**Reply by nigelkeirross** [reply]
I downloaded the 2014-08-07 version & have used 2 Libre2+. It worked fine with the latest OOP2. From memory at the beginning it still said 14.5 days to go, but in the end it lasted 15.5 days.

**Reply by Diabeticliberty** [reply]
Thank you for your reply. I have ordered a sensor from Abbott as they.arw currently offering a single unit for users on trial. Love the new ones or hate them all users will be forced onto them as Abbott withdraw the Libre 2 next year. My experiences with Libre 2 and those of many users I know have not been particularly positive. The last one I changed died after 4 hours. The consequent one started stuttering after 3 hours but then limped back into life and is still attached to my arm. Maybe 2+ will be more durable but I certainly won't be holding my breath.

**Reply by parapenT1sta** [reply]
My real issues with Libre 2 are no official calibrations and all, repeat ALL sensors give me -40mg/L no matter where I insert them.

**Reply by Diabeticliberty** [reply]
I count myself as incredibly fortunate not to have a HCP who thinks that the earth is flat. I can be fully frank with him about what I am doing and how and in return he will offer mild corrections if my own decisions have no basis in clinical fact. He does however fully appreciate  that I have not got my HbA1c below 40 using blind luck and snake oil. He once asked me why Xdrip+ and not Librelink. I explained about the lack of calibration facility, the non customisable alarms with Librelink. The lack of a facility to have a realtime watch face. The fact that Librelink will not go into a workable widget on an Android or IOS phone. Most important of all of course the many drop outs between Librelink and sensor at 3.00am in a morning that most users sadly encounter. Some of these having quite severe hypo's with no warning from Librelink as it has lost connection. My GP conceded that he has many Libre users who do encounter this problem. I went on to respectfully suggest that he may with the very best of intentions offer advice on routine changes for his type 1 diabetics based on flawed Libre data. He of course did not like this at all. When I showed him the A1c number that Librelink suggested which was 10 points lower than my actual HbA1c he seemed a little more ready to accept my reasoning. In my own opinion only, Libre sensors are a half reasonable mass market part solution to better diabetic management. I would not trust Librelink however to take care of my cat.

**Reply by nigelkeirross** [reply]
@parapenT1sta Did you take readings with both xDrip+ & the official Librelink app? If it also fails with the official app, have you tried calling the Libre2 support line? I havefound them helpful.

**Reply by parapenT1sta** [reply]
> @parapenT1sta Did you take readings with both xDrip+ & the official Librelink app? If it also fails with the official app, have you tried calling the Libre2 support line? I havefound them helpful.

I use only xDrip but to compare I use Libre 1 reader to activate sensor and compare values. My doctor's team are OK with Nightscout information.

**Comment by Diabeticliberty** [comment]
I received a free sample of the 2+ from Abbott yesterday. They are currently offering a single unit on free trial. My current sensor dies on Saturday morning and I will be changing it. I will stick the 2+ on and post a reply here about how I find it.

**Comment by Diabeticliberty** [comment]
Fitted my first Libre 2+ sensor this morning. Got nothing for the first hour after the warm up. Went through all setting to ensure Xdrip+ was still correctly configured. I then updated the OOP2 application and things appear to be running OK which is of course a mild relief. I will keep this thread open for the next 2 weeks and post on how I find the reliability of the sensor and to allow anyone wishing to comment the opportunity to do so. I will then close it.

**Reply by nigelkeirross** [reply]
@Diabeticliberty Sounds good. Does it say it is expiring in 14 or 15 days time?

**Reply by Diabeticliberty** [reply]
It is showing 14 days which given the supposed 15 day life seems odd. I will see if anything changes regarding this and report back. I will confess that I am not personally overly concerned about the 14 or 15 day thing. I just need the sensors to work with Xdrip+.

**Reply by parapenT1sta** [reply]
Hopefully xDrip+ doesn't stop the sensor at 14.5 days like Libre 2 sensors.

**Reply by PhilTU** [reply]
My L2+ ran 15.5 days with xDrip. It recorded  sensor age correctly but I pay no attention to "days left" - there's a fair chance this is hard coded and needs revision for the arrival of Libre 2plus.

**Comment by Diabeticliberty** [comment]
Xdrip+ has been giving readings all afternoon which are miles away from fingerprick tests. Calibrations have had no effect whatsoever. Then an hour ago I have lost everything and think that the sensor may be dead. I was worried about something like this happening. It does not bode at all well for me with Libre 2+ and Xdrip+ going forward.

**Reply by nigelkeirross** [reply]
@Diabeticliberty If you are concered about the sensor, one thing you can do is to use the official LibreLink app to get a reading from it. It is also most convenient if this is installed on a different phone to the one you use for xDrip+. Do you have an old phone, or a friend/partner who has a phone you can install it on? This is the best way to find out if the sensor is faulty or otherwise. Use this - NFC tap on the sensor. Remember, after that, you will need to then do an NFC tap with your xDrip phone to resume readings on that phone. 
Also, are you getting any strange comments in the "View Events Log" ? (3 dots on top right)

**Reply by nigelkeirross** [reply]
@Diabeticliberty That is good that you are back up & running. The sensor age going backwards or not advanced is a very misleading message. This happens when the sensor is giving bad readings, and xdrip deletes these readings from its internal list. It then looks back to find the last good reading, and it finds that it is at the same time as the last reading, so it says "sensor age has not advanced". It really would be better if it said "bad readings from sensor deleted". I strongly recommend looking into the LibreLink app. That is a very good way of helping determine if it is a dodgy sensor, or something else going on.

**Reply by Diabeticliberty** [reply]
Thank you for the heads up. I am always a little reticent about reinstalling Librelink once I have deleted it after starting a sensor. While I fully take on board your suggestion and welcome your help the thought of Librelink grabbing the Bluetooth connection to the sensor from Xdrip+ makes me very uneasy. My hope is that everything settled down. I will report back with progress or lack thereof.

**Comment by Diabeticliberty** [comment]
The event log showed issues with sensor age not advancing. I think maybe this has something to do with the 15 day age of the Libre 2+ but cannot be completely sure. In the end I deleted Xdrip+ and did a fresh install. The sensor is now reading again and Xdrip+ is moving data close enough to fingerprick blood tests. I hope that in 12 hours time I am not back with the same issues but fear I may well be.

**Comment by Diabeticliberty** [comment]
Xdrip+ and sensor have now been running for approximately 18 hours with no further negative issues. In all fairness I am having similar problems every time I fit a new sensor. Runs for a few hours seemingly OK then reading drops over the course  of about 90 minutes until the display just shows LOW then there seem to be communication issues and either the sensor just dies or Xdrip+ requires a full install.  Running now though so I am relieved.

**Reply by nigelkeirross** [reply]
@Diabeticliberty Good to hear it now seems stable. If you are regularly having problems when a sensor is new, it might indicate calibration issues. Firstly, I always apply the sensor and start it ( with LibreLink or the abbott device if you use is) and let it run for over 4 hours to properly warm up before I switch from old to new. Secondly, make sure that your blood sugar is both stable (not rising/falling steeply), and in a "normal" range - not too high or low. Remember the CGM is about 20 mins delayed from your finger prick reading.
When xDrip displays "Low" this does not indicate a functional problem with xDrip, just that the reading is below a certain level (around 2.x in MMOL) - it continues to send the actual readings to the looper app, or any other app & upload to Nightscout if you have that set up. I would suggest that when this happens, you do another finger prick & calibrate the sensor again in xDrip. Certainly do this before re-installing xDrip.

**Reply by Diabeticliberty** [reply]
Been using Libre since the 2nd week it became available to UK diabetics. Was self funding for ever before prescriptions. I know a lot of users wait for varying amounts of time before running with the new one but personally I have never found the need. Having said that though with my current issues you may well have an extremely relevant point. As regards your second point which is calibration I always run about half a dozen calibrations through Xdrip+ in the first 24 hours of a new sensor. I calibrate until the manually inputted figure stops changing the value on display.

**Comment by Diabeticliberty** [comment]
I am now 15 days into this sensor. After it's rather wobbly start things calmed right down. It has presented no issues at all other than those previously mentioned. I would add that random fingerprick tests have shown it to be the most accurate sensor I have ever ran with. This afternoon I am back to using a standard Libre 2 but plan to try to swap the ones I currently have if my pharmacist is in a good mood tomorrow. I will add that after inserting a new sensor this afternoon I will leave it for 4 or 5 hours after insertion following the excellent advice from @nigelkeirross I have never found the need to wait with an unstarted sensor but given recent issues it can't do any harm.

**Reply by nigelkeirross** [reply]
@Diabeticliberty Lets hope the new one works well. Fingers both crossed & pricked :-)

**Comment by Diabeticliberty** [comment]
OK folks 15.5 days and my sensor is now dead. It seems an appropriate time to close this thread. Thanks to all who contributed.

**Comment by Diabeticliberty** [comment]
Just reopened this to advise that my pharmacist has swapped a couple of Libre 2's for 2+ and my GP has changed my prescription. All in all it has been a really easy process and the sensors do seem to be far more accurate although I do hope that suggesting this after trialling just a single unit may not be a leap too far. In any case the 2's are being phased out in August 25 so like the new ones or lump them we are all going to be stuck with them.
